app/themes/ukis/templates/introduction.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'introduction.html' %}
+
+{% set legal_text = "Your response to this survey is of great value to government, whether or not you carry out innovation activities" %}
+
+{% block header %}
+  {% include 'partials/header.html' %}
+  {% with is_hero = true %}
+    {% include theme('partials/topbar.html') %}
+  {% endwith %}
+{% endblock %}
+
+{% block business_details %}
+  <div class="u-mb-m">
+    {% if meta.survey.region_code == 'GB-NI' %}
+      <img src="{{ theme_static('logo-dof.png') }}" alt="Department of Finance" width="183" />
+    {% else %}
+      <img src="{{ theme_static('logo-beis.png') }}" alt="Department for Business, Energy and Industrial Strategy" width="660" />
+    {% endif %}
+  </div>
+
+  {{super()}}
+
+{% endblock business_details %}
+
+{% block information_to_provide %}
+
+  <div>
+    <h2 class="saturn">Basis for completion</h2>
+    <ul>
+      <li>Please complete this voluntary questionnaire for <strong>{{ meta.respondent.address.name}}</strong>.</li>
+      <li>If this business is part of an enterprise group, please answer all questions for this business in the UK only.</li>
+      <li>Do not include results for subsidiaries or parent enterprises.</li>
+    </ul>
+  </div>
+
+  <div class="panel panel--simple panel--info u-mb-s">
+    <div class="panel__body">
+      <h2 class="neptune">Definition of innovation</h2>
+      <ul>
+        <li>Innovation, for the purpose of this survey, is defined as new or significantly improved goods or services as well as processes used to produce or supply all goods or services that Company name has introduced, regardless of their origin.</li>
```

Comment:
`new`, `significantly improved goods or services` and `processes` should be in bold